### PR TITLE
Cu 866artb3p test cu GitHub mohammed emadeldin

### PR DESCRIPTION
**Pull request by Moemad-711**

### CODE REVIEW

**Comment by Moemad-711** [comment]
Task linked: [CU-866artb3p Test cu github](https://app.clickup.com/t/866artb3p)

File: Jenkinsfile [hunk 1]
```diff
@@ -24,6 +24,8 @@ pipeline {
             echo 'This will run only if the state of the Pipeline has changed'
             echo 'For example, if the Pipeline was previously failing but is now successful'
         }   
+
+
```
Comment (Moemad-711) [inline Jenkinsfile]: #866artb3p[ToDo]

**Review by Moemad-711 — COMMENTED** [review]
CU-866artb3p[ToDo]

**Review by Moemad-711 — COMMENTED** [review]
CU-866artb3p[todo]

**Review by Moemad — CHANGES_REQUESTED** [review]
CU-866artb3p[todo]

File: Jenkinsfile [hunk 2]
```diff
@@ -24,6 +24,7 @@ pipeline {
             echo 'This will run only if the state of the Pipeline has changed'
             echo 'For example, if the Pipeline was previously failing but is now successful'
         }   
+
```
Comment (Moemad-711) [inline Jenkinsfile:28]: asd

Comment (Moemad-711) [reply Jenkinsfile:28]: adearaer

**Review by Moemad — APPROVED** [review]

**Review by Moemad — APPROVED** [review]

**Pull request merged**